dev/TeachingTip/TestUI/TeachingTipPage.xaml

```diff
@@ -30,7 +30,8 @@
                                           Closing="OnTeachingTipClosing"
                                           ActionButtonClick="OnTeachingTipActionButtonClicked"
                                           CloseButtonClick="OnTeachingTipCloseButtonClicked"
-                                          Target="{x:Bind targetButton}">
+                                          Target="{x:Bind targetButton}"
+                                          FlowDirection="RightToLeft">
```

Comment:
Did you mean to have this left as RTL in the test app?

Reply:
I did, we have two tips, one in RTL one in LTR

---
In reply to: [270511702](https://github.com/Microsoft/microsoft-ui-xaml/pull/504#discussion_r270511702) [](ancestors = 270511702)